src/serialrw.rs

```diff
@@ -83,7 +91,17 @@ impl<R: AsyncRead + Unpin + Send> SerialFrameReader<R> {
             }
         };
 
+        let peer_id = self.peer_id;
+        let frame_size_limit = self.frame_size_limit();
         let mut data = vec![];
+        let mut push_data_byte = |b: u8| {
+            if data.len() > frame_size_limit {
+                Err(ReceiveFrameError::FramingError(format!("Client ID: {}, Jumbo frame threshold {DEFAULT_FRAME_SIZE_LIMIT} bytes exceeded during read.", peer_id)))
```

Comment:
Change

`format!("Client ID: {}, Jumbo frame threshold {DEFAULT_FRAME_SIZE_LIMIT} bytes exceeded during read.", peer_id)`

to

`format!("Client ID: {}, Jumbo frame threshold {} bytes exceeded during read.", peer_id, frame_size_limit)`